deps/obs-scripting/obspython/CMakeLists.txt

```diff
@@ -1,92 +1,113 @@
-cmake_minimum_required(VERSION 2.8.12)
-project(obspython)
+if(POLICY CMP0086)
+  cmake_policy(SET CMP0086 NEW)
+endif()
 
 if(POLICY CMP0078)
-	# UseSWIG generates standard target names.
-	cmake_policy(SET CMP0078 OLD)
+  cmake_policy(SET CMP0078 NEW)
 endif()
```

Comment:
Nitpick:  could these be placed in order?

Reply:
Changed.